db-service/lib/infer/index.js

```diff
@@ -97,16 +97,16 @@ function infer(originalQuery, model = cds.context?.model || cds.model) {
     if (ref) {
       const first = ref[0].id || ref[0]
       let target = getDefinition(first, model)
-      if (!target) cds.error(`"${first}" not found in the definitions of your model`)
+      if (!target) throw new Error(`"${first}" not found in the definitions of your model`)
       if (ref.length > 1) {
         target = from.ref.slice(1).reduce((d, r) => {
           const next = d.elements[r.id || r]?.elements ? d.elements[r.id || r] : d.elements[r.id || r]?._target
-          if (!next) cds.error(`No association "${r.id || r}" in ${d.kind} "${d.name}": ${d}`)
+          if (!next) throw new Error(`No association “${r.id || r}” in ${d.kind} “${d.name}”`)
```

Comment:
```suggestion
          if (!next) throw new Error(`No association "${r.id || r}" in ${d.kind} "${d.name}"`)
```

Or is there a reason for using `”`?

Reply:
Just personal taste. Also, we use those unicode quotes throughout the compiler messages, hence I used them in most error messages. With this change, I also commited to one quotation style. I dont mind using `"` instead, if you wish.

Reply:
I just stumbled over it and I think this is something we should align.

Reply:
I agree, that is why I wanted to use the same quotation style as in the compiler. Maybe we need to bring this up for a broader audience (runtimes, tooling etc). Besides that, is this change okay for now?